cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/aggregation/WindowAggregationDefinition.java

```diff
@@ -33,15 +33,30 @@ public class WindowAggregationDefinition {
   private Map<String, Expression> selectExpressions;
   private List<OrderByExpression> orderByExpressions;
   private WindowFrameType windowFrameType;
+  private String numberOfPartitions;
```

Comment:
Number of partitions is not required to generate the SQL statement. Please remove.

Reply:
Okay